sdk/cognitiveservices/cognitiveservices-luis-runtime/src/lUISRuntimeClientContext.ts

```diff
@@ -42,7 +39,7 @@ export class LUISRuntimeClientContext extends msRest.ServiceClient {
 
     super(credentials, options);
 
-    this.baseUri = "{Endpoint}/luis/v3.0-preview";
+    this.baseUri = "{Endpoint}/luis/prediction/v3.0";
```

Comment:
Is this an expected change?

Reply:
yes that's the new url for LUIS runtime v3.0